ErrorBlocks.md

```diff
@@ -0,0 +1,156 @@
+<br/></br>
+- Scale will most likely be off by a factor of 10. For example models from Solidworks will most likely be increased by 10x or 100cm to 1000 units, but this scaling changes based on the initial CAD program. You must scale every entity down or up by that factor to match units to cm.
+</p></details>
+
+#### Creation of Void/Vacuum Volume and Graveyard
+
+_A method (implicit compliment) is implemented automatically during a DAG-MCNP5 run that creates a void volume in all spaces without geometry. This means that while one could explicitly create void volumes, it is often not necessary._
+<br/></br>
+_The vacuum and graveyard volumes are not part of the model being analyzed, but play a role in the final analysis. When particles move freely through the vacuum volume and stop once they reach the graveyard volume._
+
+| Steps  | In Graphic User Interface  | In Command Line  |
+|---|---|---|
+| __Create a cube large enough to envelop all other volumes.__  | - Select __Geometry -> Volume -> Create__ and select __Brick__ from the drop down menu. <br/></br> - Type 500 in each dimension box __(width, height, depth)__ and press __Apply__ to create the block.  | - The general form is `create [shape] [dimensions based on shape]`. </br> - For a simple cube 500x500x500 you can enter `create brick x 500`.  |
+| __Create another slightly larger cube to be used for graveyard.__  | - In command panel create another brick in the same way as part 1, using 750 as the new dimension in each dimension box.  | - Enter `create brick x 750`  |
+| __Center volumes within the larger cubes.__  | To move the blocks in command panel select __Geometry -> Volume -> Transform__ and select __Move__ from the drop down menu. <br/></br> - Type "5 6" into the __Volume ID(s)__ box. Then __Select Method -> Distance__ and type 100 into the __X Distance__ box. _This distance/direction may vary._  | - General form is `move vol [volume ids] [axis] [distance along axis]` <br/></br> - Enter `move vol 5 6 x 100`.  |
+| __Hollow out large cubes to match functionality.__  | - Select __Geometry -> Volume -> Boolean__ and select __Subtract__ from the dropdown menu. <br/></br> - Select volume 6 as volume A and volume 5 as volume B. Check __Keep Originals__ and click __Apply__. <br/></br> - You can then delete volume 6 by selecting __Geometry -> Volume -> Delete__ and typing 6 into the __Volume ID(s)__ box and clicking apply. <br/></br> - Select __Geometry -> Volume -> Boolean__ and select __Subtract__ from the dropdown menu. Select volume 5 as volume A and volumes 1,2,3, and 4 as volume B. Again make sure to check __Keep Originals__ and click __Apply__. <br/></br> - Then use the delete action to delete volume 5.  | - Enter `subtract vol 5 from 6 keep` <br/></br> - Enter `delete vol 6` to get rid of the original large box. <br/></br> - Enter `subtract vol 1 2 3 4 from 5 keep` or `subtract vol 1 to 4 from 5 keep`. <br/></br> - Enter `delete vol 5`. |

```

Comment:
In the interest of user friendly methods, I would suggest these be the steps to follow in this table:

1. Create inner graveyard surface*
2. Center the inner graveyard surface around the geometry**
3. Create inner graveyard surface.***
4. Combine inner and outer graveyard surfaces to generate a single volume****

Further comments on why I suggest these as the steps and also suggestions on how to change the instructions:
* In Trelis/Cubit you can actually just create a "bounding box" rather than a normal brick. This might automatically center the box around the geometry (eliminating step 2) AND will automatically create a box large enough.
** I suggest centering the first cube before creating the second cube because it is easier to create a second cube from the first rather than try to properly move two cubes. 
*** I believe there is a way to just create a new volume from an existing volume while telling it to scale by a certain factor. This can be easily used to generate an outer surface box that is say 10% larger than the inner surface. I don't know what the exact buttons are to click to do this, but it is found in the create geometry section I believe.
**** This step is basically what you have as your "hallow out large cube" step. However, there are really two different paths one could decide to take here. The first is to just do the subtraction and ignore the "keep originals" option. By not keeping the originals, there is no explicit definition of the implicit complement. The second path one could take is using the option for "Keep Originals" which explicitly defines the implicit complement as a volume. One could do "keep originals" to check that they did things correctly and then later delete the volume if they don't want it. I think it is very important to describe in this step that the intention with "keep originals" is to explicitly define the implicit complement.

Then you can also add a column that is called the "Description" to describe what this step is doing which seems to be more what your current steps are:
1. Create a cube large enough to enclose all the volumes of the geometry.
2. Center the cube so that all other volumes are contained within the bounds of the graveyard cube.
3. Create the outer surface of the graveyard.
4. Subtract the inner surface volume from the outer to create a single graveyard volume.

Also, it is important to note (probably add to your "Note" section below) is that the volume IDs are specific to this example if all the steps are followed exactly. One should double check what their volume IDs are just in case they are different.

Reply:
Also (sorry, a lot of comments on this section): Can you create screen capture images of each of these steps? So the user can see what each of these volumes are that are being created.

Reply:
FYI... Step 3 in the first list should be "outer" not "inner"

Reply:
oops- yes